### DISCUSSION TITLE
3 CT - 3 phase

### DISCUSSION THREAD

**Discussion by firefox159** [thomase1234/esphome-fake-xemex-csmb#6] [General]
Hi - how would we approach a 3 phase setup with 3 CTs?
I also don't know the value of the power usage of the charing pole.

**Comment by thomase1234** [comment]
Hi Firefox159,
You can set the fake measured load on each of the CT clamps in the automation.

https://github.com/thomase1234/esphome-fake-xemex-csmb/blob/ddb29163f90f92eb756014d3eede0d52cb8a18a3/charge_automation.yaml#L137

This link shows you how to do it for CT3. Copy and adapt for CT1 and CT2.

Best regards,
Thomas
